web-app/src/pages/tournament/TournamentCreator.vue

```diff
@@ -14,16 +14,16 @@
         :name="1"
         :title="$t('tournament.stepper.tournamentType')"
         icon="settings"
-        :done="step > 1"
-        :error="activeTournamentType"
+        :done="activeTournamentType != null"
+        :error="step > 1 && activeTournamentType === null"
       >
-        <tournament-type-selector />
+        <tournament-type-selector :activeType="activeTournamentType" />
       </q-step>
       <q-step
         :name="2"
         :title="$t('tournament.stepper.buildTeams')"
         icon="settings"
-        :error="(activeTournamentType = null)"
+        :error="step > 2 && teams.length < 2"
         :done="step > 2"
```

Comment:
I'm not sure
error prop is true when step is grater than 2 and&&... and done prop is true when step is greate than 2 too.
Is it correct?

Reply:
this will be changed after adding 3rd step (scheduling)